teaser/src/main/res/values/developer-config.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Please put your MAPBOX_ACCESS_TOKEN here -->
+<resources>
+    <string name="mapbox_access_token">pk.eyJ1IjoieXVyaWJrIiwiYSI6ImNrN29scHRtYzBhajYzZ3A5aWdxMWw3Y3UifQ.-4b25FZ4UX22YShloZ9bWQ</string>
```

Comment:
should it be here?

Reply:
Generally, it is a bad idea to keep a token here, but I don't know the context.
@yunikkk @kiryldz  could you comment please?

Reply:
This file should not be under git.
Please take a look at `app/.gitignore` as an example.

Reply:
good catch!